routers/rooms.js

```diff
@@ -71,8 +72,13 @@ router.route("/room/:room_id/info")
         }
     });
 
+const download_limit = rateLimit({
+    'windowMs': 60 * 10 * 1000,
+    'max': 10
+});
+
 router.route("/room/:room_id/subrooms/:subroom_id/versions/:version_id/download")
-    .get(authenticateToken, async (req, res) => {
+    .get(authenticateToken, download_limit, async (req, res) => {
```

Comment:
Downloading rooms now has a rate limit due to the fact that it reads from Firebase, which has a bandwidth limit.